files/zh-cn/web/api/gamepad/index.html

```diff
@@ -51,34 +51,11 @@ <h2 id="示例">示例</h2>
 
 <h2 id="规范">规范</h2>
 
-<table class="standard-table">
- <tbody>
-  <tr>
-   <th scope="col">规范</th>
-   <th scope="col">状态</th>
-   <th scope="col">备注</th>
-  </tr>
-  <tr>
-   <td>{{SpecName("Gamepad", "#gamepad-interface", "Gamepad")}}</td>
-   <td>{{Spec2("Gamepad")}}</td>
-   <td>Initial defintion</td>
-  </tr>
-  <tr>
-   <td>{{SpecName('WebVR 1.1', '#gamepad-getvrdisplays-attribute', 'displayId')}}</td>
-   <td>{{Spec2("WebVR 1.1")}}</td>
-   <td>Defines the {{domxref("Gamepad.displayId")}} property.</td>
-  </tr>
-  <tr>
-   <td>{{SpecName("GamepadExtensions", "#partial-gamepad-interface", "Gamepad extensions")}}</td>
-   <td>{{Spec2("GamepadExtensions")}}</td>
-   <td>Defines the {{anch("Experimental extensions to Gamepad")}}</td>
-  </tr>
- </tbody>
-</table>
```

Comment:
here

Reply:
reverted in 36c5b428c23a94463782e3808c7a4ebc1268412a